gateway/handler.ts

```diff
@@ -50,24 +52,28 @@ const handleManyCookies = (headers: any = {}) => {
   return { headers };
 };
 
+let serverHandler: ReturnType<ApolloServer["createHandler"]> | null = null;
```

Comment:
moved it outside so that everything the graphqlHandler is called we do not have to create a new handler, should improve performance